recipes/Flask-Fixtures/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "Flask-Fixtures" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "2b8191f995896c0dee384b18b0ee236ab636b921856636c9ff1429c1ad10cb8b" %}
```

Comment:
No need to set a jinja variable that will be used in a single place.
Please but the checksum in the field below.

Reply:
Removed the Jinja variable and added the checksum value in the sha256 field.